### PR TITLE
add missing onTop annotation to @admin

### PR DESCRIPTION
**Pull request by wodka**
I am targeting this branch, because the change is BC.

## Changelog

```markdown
### Added
- Added `onTop` parameter on `@Admin` annotation
```

## Subject

add on_top tag functionality to annotations as seen here: https://sonata-project.org/bundles/admin/3-x/doc/cookbook/recipe_knp_menu.html

### CODE REVIEW

**Comment by jordisala1991** [comment]
I did a little modification on your changelog, to make it more clear.

**Comment by greg0ire** [comment]
No docs update needed?

File: Tests/DependencyInjection/Compiler/AnnotationCompilerPassTest.php [hunk 1]
```diff
@@ -104,6 +106,7 @@ public function testAdmin()
         $annotation->label = 'myLabel';
         $annotation->showInDashboard = false;
         $annotation->translationDomain = 'OMG';
+        $annotation->onTop = true;
```
Comment (dmarkowicz) [inline Tests/DependencyInjection/Compiler/AnnotationCompilerPassTest.php]: For docs consistency it should be also added to annotation above.
```
/*
 * @Admin(
 *      onTop=true,
```

**Comment by wodka** [comment]
also updated the documentation

**Comment by jordisala1991** [comment]
Thanks @wodka :)

**Pull request merged**